next.config.mjs

```
@@ -1,9 +1,48 @@
 import bundlerAnalyzer from "@next/bundle-analyzer";
 import packageInfo from "./package.json" with { type: "json" };
 import { createRequire } from "node:module";
+import { createClient } from '@supabase/supabase-js';
 
 const require = createRequire(import.meta.url);
 
+// Load config from database at build time and set as environment variable
+// Config is now ~2.8 KB (down from ~29 KB), so env var approach works fine
+async function loadConfigFromDB() {
+  const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL;
```

ESLint

[error] 'process' is not defined.

(no-undef)